docs/v0.4/getting-started/docker-for-windows.md

```diff
@@ -36,53 +27,67 @@ Enable Kubernetes in the Kubernetes tab, click on Apply, and wait for the instal
 
 ![enable Kubernetes](/img/docker-kubernetes-windows.png)
 
-If you previously had minikube or GKE configured, switch your kubectl context to "docker-desktop" using a PowerShell or command window.
+Confirm that your kubectl context is pointing to "docker-desktop".
+
+```sh
+kubectl config current-context
+```
+
+If you previously had a different cluster configured, switch your kubectl context to "docker-desktop" using a PowerShell or command window.
 
 ```sh
 kubectl config use-context docker-desktop
 ```
 
 ### monitor your cluster
-At this point it is useful to monitor your Kubernetes cluster. This PowerShell function will call `kubectl get` every 3 seconds.
+At this point it is useful to monitor your Kubernetes cluster. If you have [git bash](https://gitforwindows.org/) installed, create a bash script called `watch`, with the following content.
 
-```powershell
-function watchpods { while(1){ kubectl get pod --all-namespaces; start-sleep -seconds 3; clear }}
```

Comment:
why switch to bash scrit? I liked the powershell function on

Reply:
the powershell function flashes making it very hard to use copy/paste - but if you can translate this design to powershell i'll take it :)

Reply:
can't we keep both and let the user decide

Reply:
i suppose, but it's already starting to feel like too much for a getting-started.